packages/core/src/index.ts

```diff
@@ -35,7 +35,11 @@ export {
   pathToStateValue,
   toObserver
 } from './utils.ts';
-export { transition, initialTransition } from './transition.ts';
+export {
+  transition,
+  initialTransition,
+  getPotentialTransitions
```

Comment:
a rather similar utility is still exported as `__unsafe_getAllOwnEventDescriptors`, i would be good to maintain the naming scheme or make the other one "safe"

what do u want to use this for externally?